lib/edgestats.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var util = require('./util');
+var deferreds = new WeakMap();
+
+/**
+ * Get the standardized statistics for a particular MediaStreamTrack in Edge.
+ * @param {RTCPeerConnection} peerConnection
+ * @param {MediaStreamTrack} track
+ * @returns {Promise.<RTCStatsReport>}
+ */
+function edgeGetTrackStats(peerConnection, track) {
```

Comment:
@syerrapragada Nitpick: Can we rename this to `edgeGetStats()` since we are returning browser native stats?